### ISSUE TITLE
Hiding columns header when allowEmptyList is true

### ISSUE THREAD

**Issue by bahrammzm** [unclead/yii2-multiple-input#189]
is there anyway to hide columns header when `allowEmptyList => true` Or list is empty?

**Comment by InsaneSkull** [comment]
Related #152

**Comment by bahrammzm** [comment]
does not same issue

**Comment by unclead** [comment]
yep it's not the same issue and there is no such option yet, but you can add it and send us a PR 😄

**Comment by InsaneSkull** [comment]
@unclead  somewhat related, i guess can be fixed with #152. 😄

**Comment by unclead** [comment]
@bahrammzm  do you still need this improvements? 
As i understood you want to have only a button when a list of models is empty. Right?
I think it can be done on client-side without any difficulties

**Comment by unclead** [comment]
@bahrammzm up
